src/everest/strings.py

```diff
@@ -5,7 +5,7 @@
 DEFAULT_OUTPUT_DIR = "everest_output"
 DEFAULT_LOGGING_FORMAT = "%(asctime)s %(name)s %(levelname)s: %(message)s"
 
-EVEREST_SERVER = "everserver"
+EVEREST_SERVER = "fm_everserver"
```

Comment:
Should that also be renamed in the script entry point in `pyproject.toml`?

Reply:
Maybe, maybe it would be better to rename the script file instead? Might make it less confusing.

Reply:
As it sits now we are really adding a config here, as that is the ert way, though I would like to refactor this to just submit directly, and not go through the ert config which is a bit cumbersome and not very easy to read.